ceph-perf-pull-requests/config/definitions/ceph-perf-pull-requests.yml

```diff
@@ -62,6 +62,7 @@
               if type -t clang-$i > /dev/null; then
                   cxx_compiler="clang++-$i"
                   c_compiler="clang-$i"
+                  export CFLAGS=$(echo $CFLAGS | (sed -e 's/\-fstack-clash-protection//'))
```

Comment:
ACK: GCC stays unaffected.

Reply:
> yes, but seems like it is coming from system values, we are setting `CFLAGS` & `CXXFLAGS` to $(RPM_OPT_FLAGS) + extra args we can drop using 

how does `RPM_OPT_FLAGS` kick in here?

Reply:
> > yes, but seems like it is coming from system values, we are setting `CFLAGS` & `CXXFLAGS` to $(RPM_OPT_FLAGS) + extra args we can drop using
> 
> how does `RPM_OPT_FLAGS` kick in here?

@tchaikov it is being used in ceph.spec.in: 
https://github.com/ceph/ceph/blob/8b6e38a6bfdf618aa4667ede0bde43594f7172ce/ceph.spec.in#L1159

from shaman builds: 
```
RPM_OPT_FLAGS=-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection
```
```
CXXFLAGS=-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection
```

Reply:
i mean, we are not building package here. how come rpmbuild gets involved here? could you paste a link to a ceph-perf-pull-requets failure ?

Reply:
i saw the failure first time radek opened the PR https://github.com/ceph/ceph/pull/35337 but yes, in succeeding runs, I see we do not use the workflow differing from shaman. 
Will close this, until verified. 
Just one question, what is the first trigger, is it adding a label or comment that jenkins picks?

Reply:
the failure you spotted should have been fixed by https://github.com/ceph/ceph/pull/35336 .

trigger what? if you are talking about the CBT based perf test, see https://github.com/ceph/ceph-build/blob/master/ceph-perf-pull-requests/config/definitions/ceph-perf-pull-requests.yml , and please search for "trigger".